packages/utils/src/urlConstants.ts

```diff
@@ -2,10 +2,10 @@ const FAQ_BASE_URL = "https://help.civil.co";
 
 export const urlConstants = {
   // Marketing site  / Foundation
-  MARKETING_SITE: "https://civil.co",
-  REGISTRY: "https://registry.civil.co/",
-  DASHBOARD: "https://registry.civil.co/dashboard",
-  CONSTITUTION: "https://civil.co/constitution",
+  MARKETING_SITE: "https://learn.civil.co",
+  REGISTRY: "https://civil.co/",
+  DASHBOARD: "https://civil.co/dashboard",
+  CONSTITUTION: "https://learn.civil.co/constitution",
   FOUNDATION: "https://civilfound.org",
   COUNCIL: "https://civilfound.org/#civil-council",
   APPLY: "https://civil.co/join-as-a-newsroom",
```

Comment:
Ditto. I see https://civil.co/join-as-a-newsroom/ exists but https://learn.civil.co/join-as-a-newsroom/ does not exist though, not sure what the plan is for that page.